src/test/java/com/company/verifytests/UserInfoTest.java

```diff
@@ -0,0 +1,52 @@
+package com.company.verifytests;
+
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserInfoTest extends BaseTest {
+    LoginPage loginPage = new LoginPage(driver);
+    LocalDate date = LocalDate.now();
+    DateTimeFormatter format =
+            DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    String currentDate = date.format(format);
+
+    @Test
+    public void testUserInfo(){
+        loginPage = openLoginPage();
+        loginPage.loginIntoSystem("TestUser07","Vfylhfujhf!1");
```

Comment:
`LoginPage loginPage = new LoginPage(driver);` why do you need this initialization? And after that you reassign value for loginPage in row 20.

Reply:
Fixed